.pipelines/containerSourceData/scripts/BuildGoldenContainer.sh

```diff
@@ -168,6 +170,11 @@ function validate_inputs {
         exit 1
     fi
 
+    if [[ ! -f $TOOLCHAIN_RPMS_TARBALL ]]; then
+        echo "Error - No RPMs tarball found."
```

Comment:
Ideally please mention also which tarball we're missing and print the value of `TOOLCHAIN_RPMS_TARBALL` to ease debugging.